spec/services/encrypted_key_maker_spec.rb

```diff
@@ -57,6 +61,16 @@
 
         expect(subject.unlock(user_access_key, encryption_key)).to eq hash_E
       end
+
+      it 'raises exception when params are invalid' do
+        expect do
+          subject.unlock(user_access_key, '!@#$%^&*()')
+        end.to raise_error Pii::EncryptionError
+
+        expect do
+          subject.unlock('foobar', 'zzzz')
+        end.to raise_error Pii::EncryptionError
```

Comment:
codeclimate dislikes multi-line `{}` syntax, and most of the rest of the `raise_error` tests in `spec/` use `do/end`. So I thought that was the standard.